README.md

```diff
@@ -1,26 +1,109 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+
+let num = 11;
+while (num > 1) {
+  console.log(num)
+  num -=1
+}
+
+for (let num = 13; num >= 2; num -=1) {
+  console.log(num)
+}
+
+2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
 For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
+
+let num = 0;
+while(num <= 20) {
+  if(num % 2 === 0) {
+    console.log(num + ' is even')
+  } else {
+    console.log(num + ' is odd')
+  }
+  num++
+}
+
+for (let num = 0; num < 11; num +=1) {
+if(num % 2 === 0){
+  console.log(num + ' is even')
+} else {
+  console.log(num + ' is odd')
+}
+}
+
+
+
+3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num".
 For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
 
-_Bonus_ think of another way to solve it. 
+let num = 0;
+while(num <= 10) {
+  console.log( num * 9)
```

Comment:
The only thing here is that you're supposed to log the entire math equation. What you're doing mathematically and loop-wise, however, is great :)